### DISCUSSION TITLE
Which messages indicated problems in Ruff?

### DISCUSSION THREAD

**Discussion by qarmin** [astral-sh/ruff#3917] [Q&A] [answered]
Currently in fuzzer when I running ruff with --fix, when in output is one of this messages
```
Failed to create fix
RUST_BACKTRACE
This indicates a bug in
```
I mark this file as broken
Is there any other way which basing on output to find file that cause problems?

**Comment by charliermarsh** [comment]
Not right now, but we should make that possible, you're right.

**Comment by charliermarsh** [comment, marked as answer]
Transferring to an issue since there's work to do here.
